Extension/package.json

```diff
@@ -973,6 +973,12 @@
                             ]
                         },
                         "scope": "resource"
+                    },
+                    "C_Cpp.reduceRecursiveIncludes": {
+                        "type": "boolean",
+                        "default": true,
+                        "markdownDescription": "%c_cpp.configuration.reduceRecursiveIncludes.markdownDescription%",
+                        "scope": "window"
```

Comment:
Shouldn't this have an enum like

```
                        "enum": [
                            "default",
                            "enabled",
                            "disabled"
                        ],
```

I'm not sure if "default" is needed (we could potentially use that to auto-set the default from an experiment).

Reply:
Oh -- it's a bool -- your description references "enabled"/"disabled"? Should that be `true`/`false` instead?